### DISCUSSION TITLE
Post to CRUD service returns error to calling service but not in port forward

### DISCUSSION THREAD

**Discussion by theliuk** [mia-platform/community#292] [Q&A] [answered]
### Which section?

Console

### Which is the version?

v11

### Description

We have a service that is used to call the crud service, but now we are noticing a strange problem.
When the service tries to post an entity onto the crud, the crud answers like this:

**status**: 404
**payload**:
```json
{
  "error": "not found oas definition: POST /prv_dat_san_practitioners_view",
  "message": "The request doesn't match any known API",
  "statusCode": 404
}
```

We noticed this thanks to Mizu.

The strange part is that if we call the crud via port forward, the same request with the same body goes well (as expected).

The calling service uses the custom plugin lib version **v4.1.1**.
CRUD service: **v5.4.2**

**Comment by davidebianchi** [comment, marked as answer]
Hi @theliuk! Do you use Rond? Because this seems to be the error message when Rond does not know the called API, so probably you haven't declared the specific API in rond

**Reply by theliuk** [reply]
I do have ROND on the crud service, and there wasn't a definition for that endpoint. That may be the problem. I'll try and I'll let you know.

**Reply by theliuk** [reply]
It was ROND :D 
![image](https://github.com/mia-platform/community/assets/809858/c60bb1fc-e287-46e9-b589-12b1477895b6)

**Reply by fredmaggiowski** [reply]
It was you not configuring rond, don't blame the tool 😜

**Reply by theliuk** [reply]
Indeed, I was crying because of me.
